catalog/task/create-github-release/0.1/create-github-release.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  description: >-
+    Tekton task that creates a release on github.com via the GitHub API
+  params:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+    - name: content_directory
+      type: string
+      description: "The directory inside the workspace to find files for release"
+      default: "release/"
+  workspaces:
+    - name: data
+      description: The workspace where the binaries to release reside
+  results:
+    - name: url
+      type: string
+      description: URL to inspect the created release
+  steps:
+    - name: create-release-from-binaries
+      image: registry.access.redhat.com/ubi9/ubi:latest@sha256:351ed8b24d440c348486efd99587046e88bb966890a9207a5851d3a34a4dd346
+      script: |
+        #!/usr/bin/env bash
+        curl -L \
```

Comment:
I wonder if it would make sense to add the gh binary to our shared [release-utils image](https://github.com/redhat-appstudio/release-service-utils/tree/main) and use that instead of fetching it on each tak run. But this can definitely be done later.

Reply:
We need to think about how to keep that gh binary updated over time too and when to pull in new versions of it.

Reply:
True, but that problem exists regardless of where we fetch it. And also, the same can be said for any of the other tools we install and use (or the ubi9 image reference above) and we don't have a solution for those either. But I'm not saying it's ok. (In SP, we used to mostly just use the latest everywhere and fix things if they break. On this project, it seems pinning to specific versions is preferred. Both have cons and pros.)

Any suggestions how to fix this problem? :)

Reply:
I don't have a suggestion on how to fix it, but I don't think latest is a great idea. It has bit us in the past and the consequence is breaking the entire e2e suite, which is pretty bad IMO

Reply:
Resolving this. Can always be revisited if needed, I think it is fine for now